### DISCUSSION TITLE
More examples. Especially on how to use htmx, Alpine.js and Shoelace

### DISCUSSION THREAD

**Discussion by legout** [pelme/htpy#34] [General]
Hi,

first of all, thanks for creating this great library. As a (mature) python developer having very little experience in web/frontend/html development, htpy helps me a lot to get into this topics.

In some of your comments in this discussion, I found, that you gyus are using htpy with htmx, Alpine.js and Shoelace. Do you have some examples on how to use these in htpy? Maybe you can add some examples into your [Common Patterns](https://htpy.dev/common-patterns/) section.

BTW, I´ve added htpy to my [py->html list](https://github.com/stars/legout/lists/py-html). I´ve tried most of the libs on this list and so far, I like htpy most. Closely followed by domonic, dominate, uidom and html5tagger.

Thanks and regards,
legout

**Comment by pelme** [comment]
Hi! Thanks for the suggestion! I think it would be nice to have a bunch of examples with different frameworks. Do you have any specific ideas on what you would like to see?

**Comment by legout** [comment]
Hi.
here is, what might be interesting (at least for me ;-)):
 - How to include (javascript) and use libraries like htmx, Alpine.js, Shoelace or even jQuery with htpy
 Note: I am not very familiar with frontend development (html, css and javascript).
